### PR TITLE
DataGrid mouse handler: Expose pressData for subclasses

### PR DESCRIPTION
**Pull request by martinRenou**
This is useful for subclasses to know what is currently happening

### CODE REVIEW

**Comment by martinRenou** [comment]
The CI failure looks unrelated, it might pass if launched again.

File: packages/datagrid/src/basicmousehandler.ts [hunk 1]
```diff
@@ -599,28 +599,36 @@ class BasicMouseHandler implements DataGrid.IMouseHandler {
...
+   * Convert a resize handle into a cursor.
+   */
+  cursorForHandle(handle: ResizeHandle): string {
+    return Private.cursorMap[handle];
+  }
+
+  /**
+   * Get the current pressData
+   */
+  get pressData (): PressData.PressData | null {
```
Comment (blink1073) [inline packages/datagrid/src/basicmousehandler.ts:612]: Typo: space before the paren

**Review by blink1073 — APPROVED** [review]
Looks good, thanks!  Just one nit

**Comment by martinRenou** [comment]
Thanks for the review!

**Comment by blink1073** [comment]
@lumino/datagrid: 0.17.0 => 0.18.0

**Pull request merged**